Project.toml

```diff
@@ -4,6 +4,9 @@ authors = ["Jan Swierczek-Jereczek <jan.jereczek@gmail.com>", "George Datseris <
 version = "0.1.0"
 
 [deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
```

Comment:
When you add packages to the [deps] you also need to add versions to the [compat]